website/docs/utilities/interactive/partials/code/how-to-use.md

```diff
@@ -10,11 +11,17 @@ Invocation of the component would look something like this:
 
 In this case, since no `@href` or `@route` argument is provided it will generate in output an HTML `<button>` element.
 
-_Notice: a `type="button"` HTML attribute is applied by default to the element, but this can be overwritten using the "splattributes"._
+!!! Info
 
-#### With "@href" parameter (`<a>`)
+`type="button"` HTML attribute is applied by default to the element, but this can be overwritten using the "splattributes".
+!!!
 
-**🚨 ATTENTION**: we can't support the direct use of the `href` HTML attribute because we need to rely on the `@href` Ember argument to differentiate between different types of generated output.
+### With `@href` parameter (`<a>`)
+
+!!! Critical
+
+We can't support the direct use of the `href` HTML attribute because we need to rely on the `@href` Ember argument to differentiate between different types of generated output.
```

Comment:
```suggestion
We can’t support the direct use of the `href` HTML attribute because we need to rely on the `@href` Ember argument to differentiate between different types of generated output.
```

Reply:
🤔  what is the difference between these characters? The one I have is the `'` key on my keyboard.